orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import javax.annotation.Nonnull;
+import lombok.Value;
+import lombok.experimental.Accessors;
+
+/** Represents a chat message as 'assistant' to the orchestration service. */
+@Value
+@Accessors(fluent = true)
+public class AssistantMessage implements Message {
+
+  /** The role of the assistant. */
+  @Nonnull public static final String ROLE = "assistant";
```

Comment:
(Question/Minor)

Why is this public?

Reply:
I am assuming, it might be useful for the user to access the corresponding role. (No concrete use case I can think of atm)